### ISSUE TITLE
Pie Chart view size does not fit chart content

### ISSUE THREAD

**Issue by bitvale** [PhilJay/MPAndroidChart#4711]
**Summary**
Pie Chart view size does not fit chart content if description and legend are disabled:
```
chartView.description.isEnabled = false
chartView.legend.form = Legend.LegendForm.NONE
```

![Untitled-2](https://user-images.githubusercontent.com/14936809/68280012-58443b00-007d-11ea-9699-5eb235a90375.jpg)

**Expected Behavior**
Pie Chart view size fit chart content

**Device (please complete the following information):**
 - Device: Google Nexus 5, Google Nexus 5X, Google Pixel
 - Android Version: 5, 7, 9
 - Library Version: 3.1.0

**Comment by Quentinvk** [comment]
Hey Bitvale, isn't that what    dataset.selectionShift = 30f is supposed to fix? 
If you were aware of this function, I hope it helps.
Cheers

**Comment by bitvale** [comment]
I don't think this is about selectionShift, because top space on the screen shoot is the default selectionShift (12f). And the expected behavior is the same small space (selectionShift) on the bottom.
